tests/unit/array/storage/redis/test_backend.py

```diff
@@ -0,0 +1,143 @@
+from abc import ABC
+
+import numpy as np
+import pytest
+from docarray import DocumentArray
+from docarray.array.storage.base.helper import Offset2ID
+from docarray.array.storage.memory import GetSetDelMixin, SequenceLikeMixin
+from docarray.array.storage.redis.backend import BackendMixin, RedisConfig
+
+
+class StorageMixins(BackendMixin, GetSetDelMixin, SequenceLikeMixin, ABC):
+    ...
+
+
+class DocumentArrayDummy(StorageMixins, DocumentArray):
+    def __new__(cls, *args, **kwargs):
+        return super().__new__(cls)
+
+    def _load_offset2ids(self):
+        pass
+
+    def _save_offset2ids(self):
+        pass
+
+
+type_convert = {
+    'int': b'NUMERIC',
+    'float': b'NUMERIC',
+    'double': b'NUMERIC',
+    'long': b'NUMERIC',
+    'str': b'TEXT',
+    'bytes': b'TEXT',
+}
```

Comment:
Since we also have `bool` now, should it be here as well? Additional test case needed?

Reply:
I add test case test_redis_category_filter in /docarray/tests/unit/array/mixins/test_find.py